Mollie.WebApplicationCoreExample/Models/DecimalPlacesAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+namespace Mollie.WebApplicationCoreExample.Models
+{
+    public class DecimalPlacesAttribute : ValidationAttribute
```

Comment:
There are two `DecimalPlacesAttribute`, one in the `Mollie.WebApplicationCoreExample.Models` namespace and one in the `Mollie.WebApplicationCoreExample.Framework.Validators` namespace. The latter one is probably a better place.